resources/function_help/json/$vertex_Z_value

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "$vertex_Z_value",
```

Comment:
I guess it should say `$vertex_z` + also below in the example + rename the whole file?